src/ccl_cls.c

```diff
@@ -451,7 +451,7 @@ static void clt_init_wL(CCL_ClTracer *clt,ccl_cosmology *cosmo,
       ccl_cosmology_set_status_message(cosmo, "ccl_cls.c: clt_init_wL(): error computing lensing window\n");
     }
   }
-
+  
```

Comment:
i'd be nice to avoid these whitespace changes.

Reply:
sorry, my emacs sometimes is a bit stupid. Fixed